security/s2a/testdata/README.md

```diff
@@ -0,0 +1,19 @@
+## Key Generation
```

Comment:
I think we can restructure this section as follows:
1. CA key and cert generation.
2. Client key and cert generation.
3. Server key and cert generation.

Also, please the exact command that you used to generate the files. For example, use the exact file name that you used instead of `<FILE_NAME>`. This way the instructions can directly reflect the files that you are adding in this directory.